nnstreamer_example/custom_example_opencv/nnstreamer_customfilter_opencv_average.cc

```diff
@@ -0,0 +1,143 @@
+/**
+ * NNStreamer OpenCV Custom Filter Example: Average
+ * Copyright (C) 2018 Sangjung Woo <sangjung.woo@samsung.com>
+ *
+ * LICENSE: LGPL-2.1
+ *
+ * @file  nnstreamer_customfilter_opencv_average.cc
+ * @date  25 Oct 2018
+ * @brief  OpenCV Custom NNStreamer Filter Example: Average
+ * @author  Sangjung Woo <sangjung.woo@samsung.com>
+ * @bug  No known bugs
+ * @see  nnstreamer_customfilter_example_average.c
+ * 
+ * This example calculates the average value of input tensor for 
+ * each channel (i.e. R, G & B). The shape of the input tensor is
+ * [N][y][x][M] and that of the output tensor is [N][1][1][M]. 
+ */
+
+#include "opencv2/opencv.hpp"
```

Comment:
```suggestion
#include <opencv2/opencv.hpp>
```

However, this is very trivial thing.

Reply:
@again4you PTAL.